src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,27 @@
-export const PostInfo = () => <>Put the post here</>;
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+import './PostInfo.scss';
+
+export const PostInfo = ({ post: { title, user, body, comments } }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{title}</h3>
+
+      <p>
+        {' Posted by  '}
+        <UserInfo user={user} />
+      </p>
+    </div>
+    <p className="PostInfo__body">{body}</p>
+    {comments.length ? (
+      <CommentList comments={comments} />
+    ) : (
+      <>
+        <hr />
+        <b data-cy="NoCommentsMessage">No comments yet</b>
+      </>
+    )}
+    ;
```

Comment:
```suggestion
```

Reply:
This is the merit of the automatic prettier and the linter accepts it, who am I to argue with them)))?